portfolio/src/main/webapp/imagegallery.html

```diff
@@ -0,0 +1,49 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <meta charset="UTF-8">
+    <meta content="width=device-width, initial-scale=1.0">
+    <title>My Favorite Images</title>
+    <link rel="stylesheet" href="style.css">
+  </head>
+  <body>
+    <div id="content">
+      <h1>Image Gallery</h1>
+      <nav>
+        <a href="index.html">About Me</a>
+        <a href="imagegallery.html">Image Gallery</a>
+        <a href="hiddentalent.html">Hidden Talent</a>
+        <a href="https://www.linkedin.com/in/jamieatucker/">LinkedIn</a>
+        <a href="https://github.com/Jamieatucker">Github</a>
+      </nav>
+      <p>When I was younger, I did not see the appeal of taking pictures because I was worried about them taking up too much space on my 8 GB iPod. As a result, I don't have many pictures of
+        myself or others until freshman year of high school (when I upgraded to an iPhone 6S).
+      <p>But now, I enjoy taking pictures. Pictures provide a sense of nostalgia for me when I look back at them years down the road. They serve as a memento of what life was back then,
+        and even can serve as a message to reach out to an old friend I haven't talked to in years. It's not about how great the picture is, but how great the moment during the picture was.
+      <p>Here are a few of my favorite photos I've taken throughout my life:</p>
+      <figure>
+        <img src="/images/beachday.jpg" alt="Picture of me water-pedalling on the lake." class="center">
+        <figcaption>Pedal to the Water</figcaption>
+      </figure>
+      <figure>
+	    <img src="/images/osustadium.jpg" alt="Picture of Ohio State football stadium at night." class="center">
+        <figcaption>Ohio State Vs. Michigan State: 2019</figcaption>
+      </figure>
+      <figure>
+        <img src="/images/goldenstate.jpg" alt="Picture of the inside of the Golden State Warriors Arena." class="center">
+        <figcaption>A Visit to see the Golden State Warriors Play in 2016!</figcaption>
+      </figure>
+      <figure>
+        <img src="/images/osumspeap.jpg" alt="Picture of my friends and I during the OSU Morrill Scholarship Program Early Arrival Program." class="center">
+        <figcaption>Ohio State Morrill Scholars Early Arrival Program: 2019</figcaption>
+      </figure>
+      <figure>
+        <img src="/images/jtcompsci.jpg" alt="Picture of me with computers in the background." class="center">
+        <figcaption>A Persona of Me Encapuslated in this Picture.</figcaption>
+      </figure>
+      <footer>
+        <a href="index.html" id="pvpage"> Previous</a>No copyright infringement is intended. <a href="hiddentalent.html" id="nxpage"> Next </a>
+      </footer>
+    </div>
+    </body>
```

Comment:
I think this is two spaces too indented.

Reply:
I looked into it and the </footer> tag is two spaces. So I think it's good.